### ISSUE TITLE
request shorter frame code points before RFC publication

### ISSUE THREAD

**Issue by marten-seemann** [marten-seemann/draft-seemann-quic-address-discovery#20]
A 2-byte code point would be nice.

**Comment by huitema** [comment]
I do not really understand the process either. I was expecting some kind of form, but apparently we just need to send an email to the IANA, explaining that we need a code for a transport type and also two consecutive codes for the frame types. If we have to propose values, I would go with 0xAD1 for the "Address Discovery" TP, and 0x0B5A and 0x0B5B for the observed addresses V4 and V6 (OBServed A and OBServed B), but that's just because I like mnemonics.

But I wonder whether we should wait for WG adoption... which my take a bit more time.

**Comment by marten-seemann** [comment]
I agree. We can always change code points and bump the draft version.

**Comment by marten-seemann** [comment]
@nibanks had a suggestion for a particular 2-byte code points. Would you mind posting them here?

**Comment by nibanks** [comment]
I started implementing this in msquic and I just shortened what you had to:
```c
    QUIC_FRAME_OBSERVED_ADDRESS_V4  = 0x9f81ULL,
    QUIC_FRAME_OBSERVED_ADDRESS_V6  = 0x9f82ULL,
```

**Comment by huitema** [comment]
@nibanks these numbers are larger than 2^14, and will be encoded on 4 bytes. If we want two bytes, we should use something like:
```c
    QUIC_FRAME_OBSERVED_ADDRESS_V4  = 0x1f81ULL,
    QUIC_FRAME_OBSERVED_ADDRESS_V6  = 0x1f82ULL,
```

**Comment by nibanks** [comment]
Ah, you're right. I'm fine with those values you suggest.

**Comment by marten-seemann** [comment]
@LPardue How do you suggest we request the code points? Should we just pick two? Or just we write a note to the RFC editor / IANA?
